README.md

```diff
@@ -96,6 +121,12 @@ educational resources, including our own QGreenland User Guide.
 * [QGreenland official documentation](https://qgreenland.readthedocs.io)
 
 
+# Troubleshooting
+
+See our user troubleshooting guide
+[here](https://qgreenland.readthedocs.io/en/latest/user/how-to/troubleshooting.html).
+
+
 # Contributing
 
 Please see [contributing
```

Comment:
Below, we have an "Acknowledgements" section which I think can serve as the "Credit" section from the template (https://github.com/nsidc/template/blob/main/README.md#credit)